test/extended/util/framework.go

```diff
@@ -50,15 +50,20 @@ const pvPrefix = "pv-"
 
 // WaitForOpenShiftNamespaceImageStreams waits for the standard set of imagestreams to be imported
 func WaitForOpenShiftNamespaceImageStreams(oc *CLI) error {
+	fnName := "WaitForOpenShiftNamespaceImageStreams"
 	langs := []string{"ruby", "nodejs", "perl", "php", "python", "wildfly", "mysql", "postgresql", "mongodb", "jenkins"}
 	scan := func() bool {
 		for _, lang := range langs {
+			fmt.Fprintf(g.GinkgoWriter, "%v: Checking %v \n", fnName, lang)
 			is, err := oc.Client().ImageStreams("openshift").Get(lang, metav1.GetOptions{})
 			if err != nil {
+				fmt.Fprintf(g.GinkgoWriter, "%v: IS Error: %#v \n", fnName, err)
 				return false
 			}
 			for tag := range is.Spec.Tags {
+				fmt.Fprintf(g.GinkgoWriter, "%v: Checking tag %v \n", fnName, tag)
 				if _, ok := is.Status.Tags[tag]; !ok {
+					fmt.Fprintf(g.GinkgoWriter, "%v: Tag Error: %#v \n", fnName, err)
```

Comment:
you're not going to get any debugging output if ok is false.  you should print the value of ok explicitly.